src/annotationLayer.js

```diff
@@ -348,7 +358,7 @@ var annotationLayer = function (args) {
           state: geo_annotation.state.create,
           layer: this
         });
-        this.addAnnotation(m_this.currentAnnotation);
+        this.addAnnotation(m_this.currentAnnotation, null);
```

Comment:
This is done throughout the other modules, so that `none` is the special value that can be used internally to mean use the `map.gcs()` while not forcing a user to constantly pass gcs parameters when what they mostly want is the `map.ingcs()`.

I would **not** make the undefined value go to `map.gcs()`, as that is certainly not what a user wants (they want the interface gcs, not the display gcs).

Using `map_gcs` instead of `none` would be okay, but this really should be a constant defined somewhere.  In which case, I lament that we don't have one `constants` module.  The user already has to import events and actions to get constants.